lib/openstudio-standards/prototypes/common/objects/Prototype.hvac_systems.rb

```diff
@@ -3249,7 +3249,7 @@ def model_add_crac(model,
         fan.setAvailabilitySchedule(hvac_op_sch)
       else
         OpenStudio.logFree(OpenStudio::Error, 'openstudio.Model.Model', "Fan type '#{fan_type}' not recognized, cannot add CRAC.")
-        return []
+        return false
```

Comment:
Looks like YARD comment for return should be updated.

Reply:
it's behind an error message, so the return type doesn't really matter. I'm aligning with the rest of the to return false after an error.